app/src/main/java/com/android/wardrobeManager/ui/images/filters/HorizontalStripes.java

```diff
@@ -9,11 +12,11 @@
     protected Map<String, DesignFilterManager.DesignFilter> filters;
     private int thickness;
 
-    HorizontalStripes(int thickness) {
+    public HorizontalStripes(int thickness) {
         this.thickness = thickness;
     }
 
-    HorizontalStripes() {
+    public HorizontalStripes() {
         thickness = 10;
```

Comment:
change this line to
`this(10)`

https://www.geeksforgeeks.org/constructor-chaining-java-examples/